assessment.md

```diff
@@ -262,8 +262,11 @@ TODO: add description for each assessment level for this skill.
 TODO: add description for each assessment level for this skill.
 
 * Understanding and adopting best practices
-
-TODO: add description for each assessment level for this skill.
+  * :flushed: = None = Does not understand and perform best practices
+  * :worried: = Inadequate = Understands best practices, but does not perform them
+  * :neutral_face: = Adequate = Understands and adopts best practices only to a moderate amount
+  * :blush: = Good = Understands and performs best practices to a good amount
+  * :laughing: = Excellent = Perform best practices and assimilate that into all of their work very well
```

Comment:
you should remove the `None =` since you have the emojis. This way, your final bullet points would look like this.
```
* :flushed: = Does not understand and perform best practices
```
You'll have to do this for all your bullet points

Reply:
I have done so and recommitted my work